lib/job-iteration/iteration.rb

```diff
@@ -128,6 +130,11 @@ def iterate_with_enumerator(enumerator, arguments)
         return false
       end
 
+      logger.info(
+        "[JobIteration::Iteration] Enumerator found nothing to iterate! " \
+        "(times_interrupted: #{times_interrupted}, cursor_position: #{cursor_position})"
```

Comment:
Why not use tagged logging like we do everywhere? E.g. times_interrupted=#{times_interrupted} cursor_position=#{cursor_position}`?